platform/ios/test/MGLAnnotationViewTests.swift

```diff
@@ -0,0 +1,23 @@
+import XCTest
+import Mapbox
+
+class CustomAnnotationView: MGLAnnotationView {
+    
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+    
+}
+
+class MGLAnnotationViewTests: XCTestCase {
```

Comment:
I think that is possible but I also think it might be a bit weird to introduce a header and bridging in the tests because of an intratest dependency that sort of artificially requires ObjC -> Swift bridging.

Since I added the Swift test to deal with a Swift-specific issue, I renamed the Swift test in https://github.com/mapbox/mapbox-gl-native/pull/9104/commits/04d73d1ed4601f3a5c0413eedc833e68506762e4. Hopefully that'll make things clearer.